app/views/views.py

```diff
@@ -24,15 +25,26 @@ def __init__(self):
         super(QuestionsList, self).__init__()
 
     def post(self):
-        qtn_id = len(Questions)
-        qtn_id += 1
+        Question_ID = len(Questions)
+        Question_ID += 1
         args = self.reqparse.parse_args()
-        question = QuestionsModels(args['title'], args['body'],
-                                   args['tags'], qtn_id)
-        Questions.append(question)
-        return make_response(jsonify({
-            'question': question.__dict__,
-        }), 201)
+        question = validate_question_input(args['title'], args['body'],
+                                           args['tags'], Question_ID)
+
+        if isinstance(question, QuestionsModels):
+            for Question in Questions:
```

Comment:
Validating that a specific questions exists can be done before creating a new instance object of a Question. You should try to minimise unnecessary computations, if they can be negated out using validations at the start.